cmd/cli/commands/node_tasks.go

```diff
@@ -0,0 +1,88 @@
+package commands
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"time"
+
+	pb "github.com/sonm-io/core/proto"
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	nodeTaskRootCmd.AddCommand(nodeTaskListCmd, nodeTaskStatusCmd)
+}
+
+var nodeTaskRootCmd = &cobra.Command{
+	Use:     "task",
+	Short:   "Operations with tasks",
+	PreRunE: checkNodeAddressIsSet,
+}
+
+func printNodeTaskStatus(cmd *cobra.Command, tasksMap map[string]*pb.TaskListReply_TaskInfo) {
+	if isSimpleFormat() {
+		for worker, tasks := range tasksMap {
+			if len(tasks.GetTasks()) == 0 {
+				cmd.Printf("Worker \"%s\" has no tasks\r\n", worker)
+				continue
+			}
+
+			cmd.Printf("Worker \"%s\":\r\n", worker)
+			i := 1
+			for ID, status := range tasks.GetTasks() {
+				up := time.Duration(status.GetUptime())
+				cmd.Printf("  %d) %s \r\n     %s  %s (up: %v)\r\n",
+					i, ID, status.Status.String(), status.ImageName, up.String())
```

Comment:
i++ in param?

Reply:
in param for what?

Reply:
yep, you are right, go doesn't allow this